lib/child-writer/writer-proxy.js

```diff
@@ -27,14 +27,15 @@ const EXIT_CODES = require('../shared/exit-codes')
 const robot = require('../shared/robot')
 const permissions = require('../shared/permissions')
 const packageJSON = require('../../package.json')
+const units = require('../shared/units')
 
 /**
  * @summary Frequency of process heartbeat signal in milliseconds
  * @constant
  * @private
  * @type {Number}
  */
-const HEARTBEAT_TIME = 1000
+const HEARTBEAT_TIME_MS = units.MILLISECONDS_IN_A_SECOND
```

Comment:
I'm not sure about this change (and also the `60 * units.MILLISECONDS_IN_A_SECOND` one) - elsewhere in the codebase we've got values specified in ms, so perhaps this should be changed back to `const HEARTBEAT_TIME_MS = 1000` ?
Although @jhermsmeier and @jviotti will have a much better idea than me what the "standard nodeJS practice" is.

Reply:
I'd just do `60 * 1e3` and be done with it, but our linter wouldn't like that I think :)